app/Http/Controllers/CelebController.php

```diff
@@ -41,12 +43,13 @@ public function store(Request $request)
     /**
      * Display the specified resource.
      *
-     * @param  \App\Models\Celeb  $celeb
+     * @param Celeb $celeb
      * @return \Illuminate\Http\Response
      */
     public function show(Celeb $celeb)
     {
-        //
+        $single_celeb = Celeb::find($celeb);
```

Comment:
Okay so here, you are already passing the object as a parameter, which means that Laravel will already fetch the record for you. 

For this reason, you do not need to find the celeb. You can just pass the $celeb to the view.